models/cice/README.rst

```diff
@@ -0,0 +1,201 @@
+###########
+CICE README
+###########
```

Comment:
Should we say 'CICE is fully supported under Manhattan' or ...

Reply:
The content in the CICE documentation is several years old. I think we should update it in a separate pull request.